templates/index.html

```diff
@@ -44,7 +44,7 @@ <h1>SRI Hash Generator</h1>
     <section id="about" class="container">
       <h2>What is Subresource Integrity?</h2>
       <p>SRI is a new <a href="https://www.w3.org/TR/SRI/">W3C specification</a> that allows web developers to ensure that resources hosted on third-party servers have not been tampered with. Use of SRI is recommended as a best-practice, whenever libraries are loaded from a third-party source.</p>
-      <p>Learn more about <a href="https://developer.mozilla.org/docs/Web/Security/Subresource_Integrity">how to use subresource integrity</a> on MDN.</p>
+      <p>Learn more about <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity">how to use subresource integrity</a> on MDN.</p>
```

Comment:
If we leave the link as is, it will use the preferred language of the user. Isn't that what we want?

Reply:
Yeah, I thought about that too. I'll revert these 2.